src/FluentValidation/Validators/StringEnumValidator.cs

```diff
@@ -0,0 +1,41 @@
+﻿namespace FluentValidation.Validators {
+	using System;
+	using System.Linq;
+	using System.Reflection;
+	using FluentValidation.Internal;
+	using Resources;
+
+	public class StringEnumValidator : PropertyValidator {
+		private readonly Type _enumType;
+
+		private readonly bool _caseSensitive;
+
+		public StringEnumValidator(Type enumType) : this(enumType, false) {

```

Comment:
Let's get rid of this constructor as the default is now specified as `true` in the extension method.